apps/vscode-extension/src/intellisense/design-token.ts

```diff
@@ -18,20 +18,25 @@ type DesignTokenCache = {
   tokenStructure: DesignTokenVariableSet;
 };
 
-const readFile = async (filePat: string) => (await vscode.workspace.fs.readFile(vscode.Uri.parse(filePat))).toString();
+const readFile = async (filePat: string) => {
+  const content = await vscode.workspace.fs.readFile(vscode.Uri.parse(filePat));
+  return content.toString();
+};
 
 const getFilesPatternsFromProjectConfiguration = async (currentFile: string): Promise<string[] | undefined> => {
   const angularFile = vscode.workspace.findFiles('angular.json');
   const nxFile = vscode.workspace.findFiles('nx.json');
   const projectFiles = vscode.workspace.findFiles('**/project.json')
     .then((files) => files.filter((file) => !path.relative(path.dirname(file.fsPath), currentFile).includes('..')));
-  const configFiles = (await Promise.all([angularFile, nxFile, projectFiles])).flat();
+  const configFilesArray = await Promise.all([angularFile, nxFile, projectFiles]);
+  const configFiles = configFilesArray.flat();
   if (configFiles.length === 0) {
     return;
   }
 
   const walkThroughProjectFiles = (node: any, mem: Set<string> = new Set()) => {
     if (typeof node === 'object' && node !== null && !Array.isArray(node)) {
+      /* eslint-disable @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-argument -- type is checked just above */
```

Comment:
the type is not really checked, the problem is probably that the input has an `any` type